### PR TITLE
Removal of reference to Redis

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Removal of reference to Redis

# What has changed
Removed Redis reference

# Links
https://trello.com/c/g3Y6jXeB

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Redis still referenced in multiple repos](https://trello.com/c/g3Y6jXeB/235-redis-still-referenced-in-multiple-repos)

File: Pipfile [hunk 1]
```diff
@@ -19,3 +19,6 @@ pgpy = "*"
 
 [requires]
 python_version = "3.6"
+
+[packages]
```
Comment (davidmort) [inline Pipfile]: not sure the dependencies should be re-locked as part of this PR

**Review by davidmort — CHANGES_REQUESTED** [review]

File: Pipfile [hunk 2]
```diff
@@ -19,3 +19,6 @@ pgpy = "*"
...
+flake8 = "*"
```
Comment (AdamHawtin) [inline Pipfile]: This shouldn't be added here, it's already a dev level dependency (you don't need it to run the acceptance tests, just running format checking).

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: Pipfile (hunk 2 above)
Comment (LeoKHoward) [reply Pipfile]: Not too sure what happened with this as I didn't even open the Pipfile when I made the other changes so couldn't understand why this had happened!

**Review by AdamHawtin — APPROVED** [review]

**Review by AdamHawtin — APPROVED** [review]

**Review by davidmort — APPROVED** [review]

**Pull request merged**